src/remote_gym/remote_environment_management.py

```diff
@@ -11,26 +18,40 @@
     dm_env_rpc_pb2_grpc,
     spec_manager,
     tensor_spec_utils,
+    tensor_utils,
 )
 from google.rpc import code_pb2, status_pb2
 
+from remote_gym.remote_environment import RemoteArgs
+from remote_gym.repo_manager import RepoManager
 
-def start_as_remote_environment(
-    local_environment: Union[gym.Env, gymnasium.Env],
+
+def create_remote_environment_server(
+    default_args: RemoteArgs,
     url: Text,
     port: int,
     server_credentials_paths: Optional[Tuple[Text, Text, Optional[Text]]] = None,
     enable_rendering: bool = False,
 ) -> grpc.Server:
     """
     Method with which every environment can be transformed to a remote one.
-    Starts the Catch gRPC server and passes the locally instantiated environment.
+    Starts the Catch gRPC server and instantiates environments on new connections.
     Requires credentials to open a secure server port in gRPC. Needs to match the client authentication.
 
-    NOTE: Use the RemoteEnvironment class to connect to a remotely started environment and provide a gym.Env interface.
+    The entrypoint file must define a function `create_environment`:
+    ```py
+    def create_environment(enable_rendering: bool, env_id: int, **kwargs) -> gym.Env:
+        return gym.make(...)
+    ```
+    * `enable_rendering` is whether the env should render.
+    * `env_id` is a unique identifier for the environment, used for non-sharable resources.
```

Comment:
This doesn't match the example entrypoint. There we don't accept `env_id`.
Is `env_id` non-optional now or does it belong to `kwargs`?